lib/connectors/base/registry.rb

```diff
@@ -20,6 +20,18 @@ def register(name, klass)
       def connector_class(name)
         @connectors[name]
       end
+
+      def connector(name, params = nil)
+        connector_class = connector_class(name)
```

Comment:
you are shadowing `connector_class`, use another variable name (`klass` maybe)

Reply:
kk